lib/driver.js

```diff
@@ -499,6 +499,11 @@ class XCUITestDriver extends BaseDriver {
       await this.setReduceMotion(this.opts.reduceMotion);
     }
 
+    if (_.isBoolean(this.opts.reduceTransparency) && !this.isRealDevice()) {
```

Comment:
consider using isSimulator condition instead

Reply:
oops I missed that 👁️

Reply:
fixed https://github.com/appium/appium-xcuitest-driver/pull/1464/commits/d6b520cef918bc2483ae874422dc598b660337cb